package.json

```diff
@@ -2,6 +2,7 @@
   "name": "dwellingly",
   "version": "0.1.0",
   "private": true,
+  "proxy": "http://localhost:5000",
```

Comment:
We already have our URL served in an environment variable, which Heroku will be transforming to a different URL for other environments